quick-build.md

```diff
@@ -77,3 +77,15 @@ To read hilog output:
 ```bash
 hdc hilog
 ```
+
+### Speeding Up Build Times
+
+Experiencing prolonged build times can be frustrating. A practical solution to mitigate this issue involves optimizing the use of Docker containers and caching mechanisms. By mounting directories for prebuilts and ccache when initiating the Docker container, you can significantly reduce build times for subsequent builds. This approach ensures that once the prebuilts are downloaded, they don't need to be fetched again, and the compilation cache is maintained across builds.

```

Comment:
I think the beginning marketing entry is not needed here. It's looks too chatgpt-like for my taste. I'd leave just the second half: "You can significantly reduce build times for subsequent builds by mounting directories for prebuilts and ccache when initiating the Docker container. This approach ensures that once the prebuilts are downloaded, they don't need to be fetched again, and the compilation cache is maintained across builds."

Reply:
thanks for the suggestion, fixed